prac_06/programming_language.py

```diff
@@ -0,0 +1,16 @@
+class ProgrammingLanguage:
+    """Represent a Programming Language object."""
+    def __init__(self, name="", typing="", reflection="", year=0):
+        """Initialise a programming language instance."""
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __is_dynamic__(self):
+        """Check whether programming language is dynamically typed or not."""
```

Comment:
I think you could condense this line? Maybe something like:
"Check if programming language is dynamic"